### PR TITLE
8.0 Change 'View Stock' button to open Quants and change it into a new style...

### PR DESCRIPTION
**Pull request by yvaucher**
... button with count

### CODE REVIEW

File: logistic_requisition/model/logistic_requisition.py [hunk 1]
```diff
@@ -585,18 +587,24 @@ def _get_total_cost(self):
...
-            'view_mode': 'tree',
-            'res_model': 'stock.location',
-            'target': 'current',
-            'view_id': False,
-            'context': {'product_id': self.product_id.id},
-            'domain': [('usage', '=', 'internal')],
-            'type': 'ir.actions.act_window',
-        }
+        action_dict = self._get_act_window_dict('stock.product_open_quants')
+        action_dict['domain'] = [('product_id','=', self.product_id.id)]
```
Comment (lepistone) [inline logistic_requisition/model/logistic_requisition.py]: missing space after the comma here breaks the build

**Comment by lepistone** [comment]
The build is broken by a minor pep8 issue. Otherwise, :+1:

**Comment by yvaucher** [comment]
Rebased

**Comment by gurneyalex** [comment]
:+1:

**Comment by foutoucour** [comment]
:+1:

**Pull request merged**